source/view.c

```diff
@@ -972,11 +1055,18 @@ static void rofi_view_refilter ( RofiViewState *state )
         tokenize_free ( state->tokens );
         state->tokens = NULL;
     }
+    if ( state->pattern ) {
```

Comment:
Yes, but in the patch it is not used. You redefine pattern in filter_elements, but state->filter is not used.